prow/cmd/phaino/local.go

```diff
@@ -246,7 +250,87 @@ func convertToLocal(ctx context.Context, log *logrus.Entry, pj prowapi.ProwJob,
 		log.Warn("Ignoring resource requirements")
 	}
 
+	// Setup gcloud credentials
```

Comment:
Document why doing this prior to other steps would be helpful

Reply:
I have adjusted the sequence and moved this after the general env vars and volumes setup.